samples/highcharts/chartchooser/continuous-distribution-scatter/demo.html

```diff
@@ -0,0 +1,8 @@
+
+<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<script src="https://code.highcharts.com/modules/boost.js"></script>
```

Comment:
I cannot see that boost is enabled in this sample, so it should not be loaded.

Reply:
Done